bodhi/client/__init__.py

```diff
@@ -79,6 +79,8 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
+	except Exception as e:
```

Comment:
It seems that this may be a tab character. We only want to use spaces in Bodhi (I should write that in the developer guide). Please set your editor to use a tabstop of 4 spaces (so that when you hit the tab button when working on Bodhi, it'll insert spaces until the next multiple of 4 is reached).